INFO 2023-09-16 13:28:16,227  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:28:25,006  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:28:25,006  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:32:55,247  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\auth_api\settings.py changed, reloading.
INFO 2023-09-16 13:32:56,140  668 Watching for file changes with StatReloader
INFO 2023-09-16 13:34:27,207  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py changed, reloading.
INFO 2023-09-16 13:34:27,808  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:34:34,956  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:34:34,957  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:35:46,924  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py changed, reloading.
INFO 2023-09-16 13:35:47,526  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:36:01,736  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:36:01,737  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:41:26,993  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:41:27,678  668 Watching for file changes with StatReloader
INFO 2023-09-16 13:42:12,458  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:42:13,125  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:42:15,317  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:42:15,319  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:44:46,073  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:44:46,694  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:44:47,253  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:44:47,254  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:46:44,227  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py changed, reloading.
INFO 2023-09-16 13:46:44,807  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:47:06,566  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:47:06,567  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:52:17,317  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:52:17,970  668 Watching for file changes with StatReloader
INFO 2023-09-16 13:52:51,107  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:52:51,755  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:53:12,835  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:53:12,836  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:53:56,965  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:53:57,554  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:54:11,026  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:54:11,027  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:55:01,473  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:55:02,071  668 Watching for file changes with StatReloader
INFO 2023-09-16 13:56:44,896  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:56:45,490  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:56:51,198  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:56:51,198  212 "POST /sendOTP/ HTTP/1.1" 500 73
ERROR 2023-09-16 13:57:14,777  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:57:14,778  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:57:46,828  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 13:57:47,418  668 Watching for file changes with StatReloader
ERROR 2023-09-16 13:57:51,972  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:57:51,973  212 "POST /sendOTP/ HTTP/1.1" 500 73
ERROR 2023-09-16 13:58:28,814  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 13:58:28,815  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 13:59:59,285  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:00:00,052  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:00:11,893  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:00:11,894  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:01:54,147  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:01:54,748  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:01:58,586  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:01:58,588  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:04:21,171  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:04:21,819  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:04:25,067  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:04:25,067  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:05:38,594  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:05:39,262  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:05:42,837  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:05:42,838  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:08:04,170  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:08:04,944  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:08:07,929  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:08:07,931  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:09:12,294  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\auth_api\settings.py changed, reloading.
INFO 2023-09-16 14:09:12,900  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:09:21,356  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:09:21,359  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:10:23,474  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\auth_api\settings.py changed, reloading.
INFO 2023-09-16 14:10:24,121  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:10:27,671  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:10:27,672  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:11:36,437  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\auth_api\settings.py changed, reloading.
INFO 2023-09-16 14:11:37,036  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:12:34,343  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:12:34,345  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:13:45,321  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\auth_api\settings.py changed, reloading.
INFO 2023-09-16 14:13:45,919  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:13:51,836  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:13:51,836  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:18:12,638  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:18:13,253  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:18:17,409  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:18:17,411  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:19:35,564  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:19:36,338  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:19:39,831  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:19:39,832  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:24:16,223  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:24:16,834  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:24:20,729  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:24:20,729  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:27:17,207  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:27:17,868  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:27:27,840  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:27:27,841  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:29:47,115  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:29:47,840  668 Watching for file changes with StatReloader
INFO 2023-09-16 14:30:43,189  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:30:43,827  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:30:48,040  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:30:48,040  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:32:04,050  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:32:04,728  668 Watching for file changes with StatReloader
INFO 2023-09-16 14:33:28,859  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:33:29,472  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:33:33,990  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:33:33,991  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:34:58,877  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:34:59,474  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:35:05,632  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:35:05,632  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:36:09,138  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:36:09,718  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:36:13,646  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:36:13,648  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:37:24,308  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:37:25,098  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:37:27,543  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:37:27,544  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 14:39:16,683  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 14:39:17,668  668 Watching for file changes with StatReloader
ERROR 2023-09-16 14:39:20,046  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 14:39:20,047  212 "POST /sendOTP/ HTTP/1.1" 500 73
WARNING 2023-09-16 15:34:17,294  241 Method Not Allowed: /sendOTP/
WARNING 2023-09-16 15:34:17,295  212 "GET /sendOTP/ HTTP/1.1" 405 40
WARNING 2023-09-16 15:34:24,748  241 Bad Request: /sendOTP/
WARNING 2023-09-16 15:34:24,749  212 "POST /sendOTP/ HTTP/1.1" 400 79
WARNING 2023-09-16 15:34:35,550  241 Bad Request: /sendOTP/
WARNING 2023-09-16 15:34:35,550  212 "POST /sendOTP/ HTTP/1.1" 400 111
WARNING 2023-09-16 15:34:55,575  241 Bad Request: /sendOTP/
WARNING 2023-09-16 15:34:55,575  212 "POST /sendOTP/ HTTP/1.1" 400 111
ERROR 2023-09-16 15:35:04,857  241 Internal Server Error: /sendOTP/
ERROR 2023-09-16 15:35:04,860  212 "POST /sendOTP/ HTTP/1.1" 500 73
INFO 2023-09-16 15:37:10,193  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\auth_api\settings.py changed, reloading.
INFO 2023-09-16 15:37:11,099  668 Watching for file changes with StatReloader
INFO 2023-09-16 15:37:16,370  212 "POST /sendOTP/ HTTP/1.1" 200 73
INFO 2023-09-16 15:40:23,295  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py changed, reloading.
INFO 2023-09-16 15:40:23,914  668 Watching for file changes with StatReloader
INFO 2023-09-16 15:40:33,923  212 "POST /sendOTP/ HTTP/1.1" 200 73
INFO 2023-09-16 15:49:42,774  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py changed, reloading.
INFO 2023-09-16 15:49:43,372  668 Watching for file changes with StatReloader
INFO 2023-09-16 15:50:13,903  212 "POST /verifyOTP/ HTTP/1.1" 200 68
INFO 2023-09-16 15:50:15,913  212 "POST /verifyOTP/ HTTP/1.1" 200 68
INFO 2023-09-16 15:50:16,978  212 "POST /verifyOTP/ HTTP/1.1" 200 68
INFO 2023-09-16 15:50:17,939  212 "POST /verifyOTP/ HTTP/1.1" 200 121
INFO 2023-09-16 16:10:56,371  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py changed, reloading.
INFO 2023-09-16 16:10:57,183  668 Watching for file changes with StatReloader
INFO 2023-09-16 16:40:43,475  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\urls.py changed, reloading.
INFO 2023-09-16 16:40:44,273  668 Watching for file changes with StatReloader
INFO 2023-09-16 16:40:58,719  668 Watching for file changes with StatReloader
INFO 2023-09-16 16:46:17,225  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\auth_api\settings.py changed, reloading.
INFO 2023-09-16 16:46:17,866  668 Watching for file changes with StatReloader
INFO 2023-09-16 16:47:47,646  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth.py changed, reloading.
INFO 2023-09-16 16:47:48,230  668 Watching for file changes with StatReloader
WARNING 2023-09-16 16:47:55,361  241 Not Acceptable: /verifyOTP/
WARNING 2023-09-16 16:47:55,363  212 "POST /verifyOTP/ HTTP/1.1" 406 67
INFO 2023-09-16 16:50:36,709  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py changed, reloading.
INFO 2023-09-16 16:50:37,274  668 Watching for file changes with StatReloader
INFO 2023-09-16 17:02:05,964  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-16 17:02:06,988  668 Watching for file changes with StatReloader
INFO 2023-09-16 17:12:17,273  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-16 17:12:18,405  668 Watching for file changes with StatReloader
INFO 2023-09-16 17:21:43,752  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-16 17:21:44,696  668 Watching for file changes with StatReloader
INFO 2023-09-16 17:26:45,752  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-16 17:26:46,337  668 Watching for file changes with StatReloader
INFO 2023-09-16 17:26:57,615  212 "POST /sendOTP/ HTTP/1.1" 200 73
INFO 2023-09-16 17:27:28,887  212 "POST /verifyOTP/ HTTP/1.1" 200 104
INFO 2023-09-16 17:27:54,360  212 "POST /registerHead/ HTTP/1.1" 200 68
WARNING 2023-09-16 17:28:06,177  241 Not Acceptable: /registerHead/
WARNING 2023-09-16 17:28:06,178  212 "POST /registerHead/ HTTP/1.1" 406 309
INFO 2023-09-16 17:28:57,017  212 "POST /registerMember/ HTTP/1.1" 200 92
INFO 2023-09-16 17:29:18,326  212 "POST /registerMember/ HTTP/1.1" 200 81
WARNING 2023-09-16 17:29:36,219  241 Not Acceptable: /registerMember/
WARNING 2023-09-16 17:29:36,219  212 "POST /registerMember/ HTTP/1.1" 406 230
WARNING 2023-09-16 17:33:37,447  241 Not Acceptable: /registerMember/
WARNING 2023-09-16 17:33:37,448  212 "POST /registerMember/ HTTP/1.1" 406 89
ERROR 2023-09-16 17:33:58,457  241 Internal Server Error: /registerMember/
Traceback (most recent call last):
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py", line 185, in post
    is_looking_for_match = string_to_bool(data.get('lookingForMatch'))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\utils.py", line 51, in string_to_bool
    if input_str.lower() == "true":
       ^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'lower'
ERROR 2023-09-16 17:33:58,461  212 "POST /registerMember/ HTTP/1.1" 500 101886
INFO 2023-09-16 17:37:42,877  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-16 17:37:43,536  668 Watching for file changes with StatReloader
WARNING 2023-09-16 17:37:51,722  241 Not Acceptable: /registerMember/
WARNING 2023-09-16 17:37:51,722  212 "POST /registerMember/ HTTP/1.1" 406 73
INFO 2023-09-16 17:38:51,620  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-16 17:38:52,232  668 Watching for file changes with StatReloader
INFO 2023-09-16 17:39:38,308  212 "GET /GETFamilyByHeadId/23/ HTTP/1.1" 200 73
INFO 2023-09-16 17:39:46,130  212 "GET /GETFamilyByHeadId/1/ HTTP/1.1" 200 363
ERROR 2023-09-16 17:40:17,284  241 Internal Server Error: /checkNumberExist/
ERROR 2023-09-16 17:40:17,285  212 "POST /checkNumberExist/ HTTP/1.1" 500 115
INFO 2023-09-16 17:46:22,821  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-16 17:46:23,388  668 Watching for file changes with StatReloader
INFO 2023-09-16 17:47:22,769  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-16 17:47:23,373  668 Watching for file changes with StatReloader
ERROR 2023-09-16 17:47:26,040  241 Internal Server Error: /checkNumberExist/
ERROR 2023-09-16 17:47:26,040  212 "POST /checkNumberExist/ HTTP/1.1" 500 115
INFO 2023-09-16 17:48:17,951  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-16 17:48:18,740  668 Watching for file changes with StatReloader
ERROR 2023-09-16 17:48:19,924  241 Internal Server Error: /checkNumberExist/
ERROR 2023-09-16 17:48:19,926  212 "POST /checkNumberExist/ HTTP/1.1" 500 115
ERROR 2023-09-16 17:49:10,886  241 Internal Server Error: /checkNumberExist/
ERROR 2023-09-16 17:49:10,886  212 "POST /checkNumberExist/ HTTP/1.1" 500 115
INFO 2023-09-16 17:51:08,484  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-16 17:51:09,061  668 Watching for file changes with StatReloader
WARNING 2023-09-16 17:51:18,492  241 Not Found: /checkNumberExist/123/
WARNING 2023-09-16 17:51:18,492  212 "POST /checkNumberExist/123/ HTTP/1.1" 404 77
WARNING 2023-09-16 17:51:57,747  241 Bad Request: /checkNumberExist/
WARNING 2023-09-16 17:51:57,748  212 "POST /checkNumberExist/ HTTP/1.1" 400 79
INFO 2023-09-17 08:20:45,164  668 Watching for file changes with StatReloader
WARNING 2023-09-17 08:20:50,981  241 Not Found: /
WARNING 2023-09-17 08:20:50,984  212 "GET / HTTP/1.1" 404 77
WARNING 2023-09-17 08:21:10,975  241 Not Found: /sendTP/
WARNING 2023-09-17 08:21:10,976  212 "GET /sendTP/ HTTP/1.1" 404 77
WARNING 2023-09-17 08:21:25,983  241 Method Not Allowed: /sendOTP/
WARNING 2023-09-17 08:21:25,984  212 "GET /sendOTP/ HTTP/1.1" 405 5674
INFO 2023-09-17 08:21:26,057  212 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,061  212 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,066  212 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,079  212 "GET /static/rest_framework/js/jquery-3.5.1.min.js HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,079  212 "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,079  212 "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,080  212 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,080  212 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,080  212 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,083  212 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2023-09-17 08:21:26,101  212 "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
WARNING 2023-09-17 08:21:26,204  241 Not Found: /favicon.ico
WARNING 2023-09-17 08:21:26,205  212 "GET /favicon.ico HTTP/1.1" 404 77
ERROR 2023-09-17 08:21:34,437  241 Internal Server Error: /sendOTP/
Traceback (most recent call last):
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py", line 28, in post
    data = request.body
           ^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\request.py", line 416, in __getattr__
    return getattr(self._request, attr)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\http\request.py", line 327, in body
    raise RawPostDataException(
django.http.request.RawPostDataException: You cannot access body after reading from request's data stream
ERROR 2023-09-17 08:21:34,443  241 Internal Server Error: /sendOTP/
ERROR 2023-09-17 08:21:34,443  212 "POST /sendOTP/ HTTP/1.1" 500 115
WARNING 2023-09-17 08:21:34,474  241 Method Not Allowed: /sendOTP/
WARNING 2023-09-17 08:21:34,475  212 "GET /sendOTP/ HTTP/1.1" 405 5674
ERROR 2023-09-17 08:21:55,175  241 Internal Server Error: /sendOTP/
Traceback (most recent call last):
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py", line 28, in post
    data = request.body
           ^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\request.py", line 416, in __getattr__
    return getattr(self._request, attr)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\http\request.py", line 327, in body
    raise RawPostDataException(
django.http.request.RawPostDataException: You cannot access body after reading from request's data stream
ERROR 2023-09-17 08:21:55,180  241 Internal Server Error: /sendOTP/
ERROR 2023-09-17 08:21:55,182  212 "POST /sendOTP/ HTTP/1.1" 500 115
WARNING 2023-09-17 08:21:55,207  241 Method Not Allowed: /sendOTP/
WARNING 2023-09-17 08:21:55,209  212 "GET /sendOTP/ HTTP/1.1" 405 5674
ERROR 2023-09-17 08:22:36,079  241 Internal Server Error: /sendOTP/
Traceback (most recent call last):
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\auth_view.py", line 28, in post
    data = request.body
           ^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\rest_framework\request.py", line 416, in __getattr__
    return getattr(self._request, attr)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\venv\Lib\site-packages\django\http\request.py", line 327, in body
    raise RawPostDataException(
django.http.request.RawPostDataException: You cannot access body after reading from request's data stream
ERROR 2023-09-17 08:22:36,083  241 Internal Server Error: /sendOTP/
ERROR 2023-09-17 08:22:36,084  212 "POST /sendOTP/ HTTP/1.1" 500 115
WARNING 2023-09-17 08:22:36,114  241 Method Not Allowed: /sendOTP/
WARNING 2023-09-17 08:22:36,115  212 "GET /sendOTP/ HTTP/1.1" 405 5674
INFO 2023-09-17 08:22:44,274  212 "OPTIONS /sendOTP/ HTTP/1.1" 200 5954
INFO 2023-09-17 08:24:35,033  212 "GET /GETAllResidents/ HTTP/1.1" 200 9297
INFO 2023-09-17 08:24:45,888  212 "GET /GETAllResidents/?format=api HTTP/1.1" 200 9341
INFO 2023-09-17 08:26:28,118  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:26:28,943  668 Watching for file changes with StatReloader
INFO 2023-09-17 08:26:34,296  212 "GET /GETAllResidents/?format=api HTTP/1.1" 200 9341
INFO 2023-09-17 08:27:58,383  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:27:59,499  668 Watching for file changes with StatReloader
WARNING 2023-09-17 08:29:10,732  241 Method Not Allowed: /checkNumberExist/
WARNING 2023-09-17 08:29:10,733  212 "GET /checkNumberExist/ HTTP/1.1" 405 5675
INFO 2023-09-17 08:29:21,841  212 "OPTIONS /checkNumberExist/ HTTP/1.1" 200 5948
INFO 2023-09-17 08:29:23,322  212 "OPTIONS /checkNumberExist/ HTTP/1.1" 200 5948
ERROR 2023-09-17 08:30:01,713  241 Internal Server Error: /checkNumberExist/
ERROR 2023-09-17 08:30:01,715  212 "POST /checkNumberExist/ HTTP/1.1" 500 115
WARNING 2023-09-17 08:30:01,788  241 Method Not Allowed: /checkNumberExist/
WARNING 2023-09-17 08:30:01,790  212 "GET /checkNumberExist/ HTTP/1.1" 405 5675
INFO 2023-09-17 08:34:06,626  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-17 08:34:07,358  668 Watching for file changes with StatReloader
WARNING 2023-09-17 08:34:16,866  241 Method Not Allowed: /checkNumberExist/
WARNING 2023-09-17 08:34:16,867  212 "GET /checkNumberExist/ HTTP/1.1" 405 5675
WARNING 2023-09-17 08:36:59,354  241 Method Not Allowed: /checkNumberExist/
WARNING 2023-09-17 08:36:59,356  212 "GET /checkNumberExist/ HTTP/1.1" 405 5675
WARNING 2023-09-17 08:38:55,164  241 Bad Request: /checkNumberExist/
WARNING 2023-09-17 08:38:55,165  212 "POST /checkNumberExist/ HTTP/1.1" 400 79
ERROR 2023-09-17 08:39:02,535  241 Internal Server Error: /checkNumberExist/
ERROR 2023-09-17 08:39:02,537  212 "POST /checkNumberExist/ HTTP/1.1" 500 115
INFO 2023-09-17 08:46:06,081  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:46:07,131  668 Watching for file changes with StatReloader
ERROR 2023-09-17 08:46:10,880  241 Internal Server Error: /checkNumberExist/
ERROR 2023-09-17 08:46:10,881  212 "POST /checkNumberExist/ HTTP/1.1" 500 95
INFO 2023-09-17 08:46:26,660  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:46:27,527  668 Watching for file changes with StatReloader
ERROR 2023-09-17 08:46:31,031  241 Internal Server Error: /checkNumberExist/
ERROR 2023-09-17 08:46:31,032  212 "POST /checkNumberExist/ HTTP/1.1" 500 93
INFO 2023-09-17 08:49:40,220  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:49:40,990  668 Watching for file changes with StatReloader
INFO 2023-09-17 08:49:45,634  212 "POST /checkNumberExist/ HTTP/1.1" 200 61
INFO 2023-09-17 08:50:13,115  212 "POST /checkNumberExist/ HTTP/1.1" 200 67
WARNING 2023-09-17 08:50:25,719  241 Not Found: /checkNumberExist/1/
WARNING 2023-09-17 08:50:25,720  212 "POST /checkNumberExist/1/ HTTP/1.1" 404 77
WARNING 2023-09-17 08:51:04,263  241 Not Found: /checkNumberExist/1/
WARNING 2023-09-17 08:51:04,265  212 "POST /checkNumberExist/1/ HTTP/1.1" 404 77
WARNING 2023-09-17 08:51:11,589  241 Unsupported Media Type: /checkNumberExist/
WARNING 2023-09-17 08:51:11,591  212 "POST /checkNumberExist/ HTTP/1.1" 415 143
INFO 2023-09-17 08:54:26,556  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:54:27,558  668 Watching for file changes with StatReloader
WARNING 2023-09-17 08:54:45,734  241 Unsupported Media Type: /checkNumberExist/
WARNING 2023-09-17 08:54:45,735  212 "POST /checkNumberExist/ HTTP/1.1" 415 143
INFO 2023-09-17 08:55:43,526  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:55:44,307  668 Watching for file changes with StatReloader
WARNING 2023-09-17 08:55:46,796  241 Unsupported Media Type: /checkNumberExist/
WARNING 2023-09-17 08:55:46,799  212 "POST /checkNumberExist/ HTTP/1.1" 415 143
INFO 2023-09-17 08:56:33,634  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:56:34,417  668 Watching for file changes with StatReloader
WARNING 2023-09-17 08:56:39,072  241 Unsupported Media Type: /checkNumberExist/
WARNING 2023-09-17 08:56:39,074  212 "POST /checkNumberExist/ HTTP/1.1" 415 143
INFO 2023-09-17 08:57:40,698  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 08:57:41,517  668 Watching for file changes with StatReloader
WARNING 2023-09-17 08:57:43,622  241 Unsupported Media Type: /checkNumberExist/
WARNING 2023-09-17 08:57:43,624  212 "POST /checkNumberExist/ HTTP/1.1" 415 143
INFO 2023-09-17 09:00:28,111  212 "POST /checkNumberExist/ HTTP/1.1" 200 61
INFO 2023-09-17 09:02:56,031  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-17 09:02:56,785  668 Watching for file changes with StatReloader
INFO 2023-09-17 09:02:59,626  212 "POST /checkNumberExist/ HTTP/1.1" 200 86
INFO 2023-09-17 09:03:19,989  212 "POST /checkNumberExist/ HTTP/1.1" 200 61
INFO 2023-09-17 09:04:16,087  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-17 09:04:16,981  668 Watching for file changes with StatReloader
INFO 2023-09-17 09:04:19,040  212 "POST /checkNumberExist/ HTTP/1.1" 200 82
INFO 2023-09-17 09:04:27,458  212 "POST /checkNumberExist/ HTTP/1.1" 200 82
INFO 2023-09-17 09:04:35,897  212 "POST /checkNumberExist/ HTTP/1.1" 200 82
INFO 2023-09-17 09:05:50,825  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-17 09:05:51,560  668 Watching for file changes with StatReloader
INFO 2023-09-17 09:05:55,638  212 "POST /checkNumberExist/ HTTP/1.1" 200 94
INFO 2023-09-17 09:12:48,914  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\auth_api\settings.py changed, reloading.
INFO 2023-09-17 09:12:49,692  668 Watching for file changes with StatReloader
WARNING 2023-09-17 09:12:57,629  241 Not Found: /checkNumberExist/1/1/
WARNING 2023-09-17 09:12:57,630  212 "POST /checkNumberExist/1/1/ HTTP/1.1" 404 2229
INFO 2023-09-17 09:13:36,877  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 09:13:37,683  668 Watching for file changes with StatReloader
WARNING 2023-09-17 09:13:40,254  241 Not Found: /checkNumberExist/1/1/
WARNING 2023-09-17 09:13:40,255  212 "POST /checkNumberExist/1/1/ HTTP/1.1" 404 2229
INFO 2023-09-17 09:14:13,911  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\middleware.py changed, reloading.
INFO 2023-09-17 09:14:14,699  668 Watching for file changes with StatReloader
WARNING 2023-09-17 09:14:19,357  241 Not Found: /checkNumberExist/1/1/
WARNING 2023-09-17 09:14:19,358  212 "POST /checkNumberExist/1/1/ HTTP/1.1" 404 78
INFO 2023-09-17 09:16:18,679  266 C:\Users\vs123\OneDrive\Desktop\gamika_user_login_api\auth_api\accounts\user_views.py changed, reloading.
INFO 2023-09-17 09:16:19,586  668 Watching for file changes with StatReloader
